var/spack/repos/builtin/packages/py-jedi/package.py

```diff
@@ -12,7 +12,8 @@ class PyJedi(PythonPackage):
     homepage = "https://github.com/davidhalter/jedi"
     url      = "https://pypi.io/packages/source/j/jedi/jedi-0.9.0.tar.gz"
 
-    # unfortunately pypi.io only offers a .whl
+    version('0.13.3', sha256='2bb0603e3506f708e792c7f4ad8fc2a7a9d9c2d292a358fbbd58da531695595b')
+    # unfortunately pypi.io only offers a .whl for 0.10.0
     version('0.10.0', sha256='d6a7344df9c80562c3f62199278004ccc7c5889be9f1a6aa5abde117ec085123',
                 url='https://github.com/davidhalter/jedi/archive/v0.10.0.tar.gz')
     version('0.9.0', sha256='3b4c19fba31bdead9ab7350fb9fa7c914c59b0a807dcdd5c00a05feb85491d31')
```

Comment:
Missing dependencies in the latest version:
```
parso>=0.3.0
python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```
You'll have to see when these requirements were added.